### DISCUSSION TITLE
Warning encountered while using vue-csv-import for vue 3

### DISCUSSION THREAD

**Discussion by Aarsh-Tatva** [jgile/vue-csv-import#121] [Q&A]
Hello folks.
Wanted to share the warning encountered which makes this npm package unusable for me. 

`Extraneous non-props attributes (value) were passed to component but could not be automatically inherited because component renders fragment or text root nodes.`

This is my code.

`<b-row class="mb-1">
              <b-col sm="6">
                <vue-csv-import
                  v-model="csvData.customers"
                  :fields="csvFields.customer"
                >
                  <vue-csv-errors />
                  <vue-csv-input :headers="true">
                    
                    
                  </vue-csv-input>
                  <vue-csv-map :autoMatch="true" />
                </vue-csv-import>
              </b-col>
              <b-col sm="3">
                <b-button
                  class="mb-1"
                  size="sm"
                  @click="downloadSample('customer')"
                  :title="$t('Download CSV Sample for Customers')"
                >
                  {{ $t('Sample CSV for Customers') }}
                </b-button>
                <br />
                <b-button
                  class="mt-2"
                  variant="primary"
                  size="sm"
                  @click="createCustomers"
                  :title="$t('Create Customers')"
                >
                  {{ $t('Create Customers') }}
                </b-button>
              </b-col>
            </b-row>`
            
I am stuck here and found nothing useful anywhere. Please share if there's a solution to it.

**Comment by BookerTlucekLw8** [comment]
Sorry to hear that you've encountered an issue with your npm package. The warning message you received can be a bit tricky to understand, but it essentially means that some non-props attributes (in this case, "value") were passed to a component, but couldn't be inherited properly because the component renders fragment or text root nodes.

Without seeing the rest of your code or the package you're using, it's hard to give specific advice on how to fix this issue. However, you might want to try looking at the component that's causing the issue and see if you can remove or rework any non-props attributes that might be causing conflicts. You could also try reaching out to the package's developer or community for more specific help.

I hope this helps! Let me know if you have any other questions or concerns

**Comment by dbilovd** [comment]
@Aarsh-Tatva were you able to find a fix for this?
